mmv1/third_party/terraform/services/container/node_config.go.tmpl

```diff
@@ -806,6 +806,14 @@ func schemaNodeConfig() *schema.Schema {
 						ForceNew: true,
 						Description: `If enabled boot disks are configured with confidential mode.`,
 				},
+				"local_ssd_encryption_mode": {
+					Type:             schema.TypeString,
+					Optional:         true,
+					Computed:         true,
+					ValidateFunc:     validation.StringInSlice([]string{"LOCAL_SSD_ENCRYPTION_MODE_UNSPECIFIED", "STANDARD_ENCRYPTION", "EPHEMERAL_KEY_ENCRYPTION"}, false),
+					Description:      `LocalSsdEncryptionMode specified the method used for encrypting the local SSDs attached to the node.`,
+					DiffSuppressFunc: tpgresource.EmptyOrDefaultStringSuppress("LOCAL_SSD_ENCRYPTION_MODE_UNSPECIFIED"),
```

Comment:
I'd just remove all references to LOCAL_SSD_ENCRYPTION_MODE_UNSPECIFIED unless your API returns it (which I very much doubt- none do that I know of). For Terraform empty/null serves the same purpose.

Reply:
Removed.

Reply:
That change seems not to have stuck! I still see it in the ValidateFunc and docs.

Reply:
Apologies, there shouldn't be any references to LOCAL_SSD_ENCRYPTION_MODE_UNSPECIFIED anymore